src/DataHandling/Logs/DataDogAudit.php

```diff
@@ -127,6 +127,27 @@ protected function getCurrentVersionElement($currentVersion, $diffElement)
         return $this->filterVersionElement($currentVersion, $diffElement);
     }
 
+    /**
+     * Method removes columns, for which changes stays only in unchanged key.
+     * @param array $diffArray subsequent elements are diff for each version
+     * @return array subsequent elements are diff for each version, filtered
+     */
+    protected function removeColumnIfOnlyUnchanged($diffArray)
+    {
+        foreach ($diffArray as $versionKey => $versionValue) {
+            foreach ($versionValue as $columnName => $columnValues) {
+                if (is_array($diffArray[$versionKey][$columnName])
+                        && isset($diffArray[$versionKey][$columnName][self::KEY_UNCHANGED])
+                        && !isset($diffArray[$versionKey][$columnName][self::KEY_ADD])
+                        && !isset($diffArray[$versionKey][$columnName][self::KEY_REMOVE])) {
```

Comment:
question: is `isset()` sufficent? So is the element never an empty array (`array()`)? Or should `!empty()` be used instead?

Reply:
Logic in getCurrentVersionElement($currentVersion, $diffElement) (especially line 117) ensures that that there is no empty array with added or removed elements (added array is unset when empty, removed array is created only if there are real elements removed).

Reply:
👍 fine in this case